Formula/rawtoaces.rb

```diff
@@ -0,0 +1,24 @@
+class Rawtoaces < Formula
+  desc "RAW to ACES Utility"
+  homepage "https://github.com/ampas/rawtoaces"
+  url "https://github.com/ampas/rawtoaces/archive/v1.0.tar.gz"
+  sha256 "9d15e7e30c4fe97baedfdafb5fddf95534eee26392002b23e81649bbe6e501e9"
+
+  depends_on "cmake" => :build
+  depends_on "aces_container"
+  depends_on "boost"
+  depends_on "ceres-solver"
+  depends_on "ilmbase"
+  depends_on "libraw"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match "Day-light (e.g., D60, D6025)", shell_output("#{bin}/rawtoaces --valid-illums", 0).strip
```

Comment:
The `, 0` is redundant with `shell_output`.

Reply:
Hi, Thanks for the suggestion!

I just removed the redundancy. Please let me know. 

Thanks!